README.md

```diff
@@ -31,12 +31,12 @@ Table of Contents
     - [E-commerce](#e-commerce)
   - [Federated Identity/Authentication](#federated-identityauthentication)
   - [Feed Readers](#feed-readers)
-  - [File Sharing and Synchronization](#file-sharing-and-synchronization)
-      - [Web based file managers](#web-based-file-managers)
+  - [File Sharing, Synchronization and Archiving](#file-sharing-and-synchronization)
```

Comment:
Please revert this change and  create a top-level section named `Document Archiving` with Paperless as it's single entry. Please also mind adding this section to the table of contents :) I don't think it should be a sub-category as archiving and synchronization software are very different in nature.